github/examples_test.go

```diff
@@ -74,3 +74,23 @@ func ExampleUsersService_ListAll() {
 		// Process users...
 	}
 }
+
+func ExamplePullRequestsService_Create() {
+	// In this example we're creating a PR and displaying the HTML url at the end.
+	client := github.NewClient(nil)
+
+	newPR := &github.NewPullRequest{
+		Title:               github.String("My awesome pull request"),
+		Head:                github.String("branch_to_merge"),
+		Base:                github.String("master"),
+		Body:                github.String("This is the description of the PR created with the package `github.com/google/go-github/github`"),
+		MaintainerCanModify: github.Bool(true),
+	}
+
+	pr, _, err := client.PullRequests.Create(context.Background(), "myOrganization", "myRepository", newPR)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	fmt.Printf("PR created: %s\n", *pr.HTMLURL)
```

Comment:
One more tiny nit... I would leave the blank line you had before this one to separate it a bit from the error above... but not a big deal either way... feel free to ignore that.

Also, when contributing to this repo in the future, which we hope you do, you don't need to force push... you can just keep adding commit after commit during the review process (which makes the changes easier for the reviewers to review), and then we squash and merge at the very end into a single commit.

Reply:
Got it. Thanks a lot!
Empty line added ;)